docs/content/en/docs/how-tos/buildpacks/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Building Artifacts with CNCF Buildpacks"
+linkTitle: "Buildpacks"
+weight: 100
+---
+
+This page describes building Skaffold artifacts with [buildpacks](https://buildpacks.io/).
+Buildpacks enable building language-based containers from source code, without the need for a Dockerfile.
+
+## Before you begin
+For this tutorial to work, you will need to have Skaffold and a Kubernetes cluster set up.
+To learn more about how to set up Skaffold and a Kubernetes cluster, see the [getting started docs](https://skaffold.dev/docs/getting-started/).
+
+To use buildpacks with Skaffold, please install the following additional tools:
+
+* [pack](https://buildpacks.io/docs/install-pack/)
+* [docker](https://docs.docker.com/install/)
+
+## Tutorial - Hello World in Go
+
+This tutorial will demonstrate how Skaffold can build a simple Hello World Go application with buildpacks and deploy it to a Kubernetes cluster.
+
+First, clone the Skaffold [repo](https://github.com/GoogleContainerTools/skaffold) and navigate to the [buildpacks example](https://github.com/GoogleContainerTools/skaffold/tree/master/examples/buildpacks) for sample code:
+
+```shell
+$ git clone https://github.com/GoogleContainerTools/skaffold
+$ cd skaffold/examples/buildpacks
+```
+
+Set the default buildpack to one that can build Go applications: 
+
+``` shell
+$ pack set-default-builder heroku/buildpacks
+```
+
+Take a look at the `build.sh` file, which uses `pack` to containerize source code with buildpacks:
```

Comment:
Can you also link the http://skaffold.dev/docs/how-tos/builders/#custom-build-script-run-locally here. I think users might need help on what `$image`  and `$PUSH_IMAGE` is.